exasol/nb_connector/cli/processing/option_set.py

```diff
@@ -104,12 +104,11 @@ def check(self) -> bool:
                 "Configuration is complete for an "
                 f"Exasol {config.backend_name} instance."
             )
-            return True
+            return
         formatted = ", ".join(missing)
         n = len(missing)
         prefix = "1 option is" if n == 1 else f"{n} options are"
-        report.error(f"{prefix} not yet configured: {formatted}.")
-        return False
+        raise ScsCliError(f"{prefix} not yet configured: {formatted}.")
```

Comment:
I think it will be useful to say which options are not configured.

Reply:
Of course it is.
This is provided by reporting the content of variable `formatted`, which contains the comma-separated list of options not configured yet.